tests/test_entities.py

```diff
@@ -78,6 +78,7 @@ def test_path_without_which(self, entity, path):
         (entities.Product, '/products', 'repository_sets/2396/enable'),
         (entities.Repository, '/repositories', 'sync'),
         (entities.Repository, '/repositories', 'upload_content'),
+        (entities.RHCIDeployment, '/deployments'),
```

Comment:
I think `(entities.RHCIDeployment, '/deployments', 'deploy')` is expected here. (Did I ask you to write something else? Sorry.)